backend/accounts/baseaccount_test.go

```diff
@@ -103,4 +109,44 @@ func TestBaseAccount(t *testing.T) {
 		require.Equal(t, EventStatusChanged, checkEvent())
 		require.Equal(t, "another test note", notes.TxNote("test-tx-id"))
 	})
+
+	t.Run("exportCSV", func(t *testing.T) {
+		export := func(transactions []*TransactionData) string {
+			var result bytes.Buffer
+			require.NoError(t, account.ExportCSV(&result, transactions))
+			return result.String()
+		}
+
+		const header = "Time,Type,Amount,Unit,Fee,Address,Transaction ID,Note\n"
+
+		require.Equal(t, header, export(nil))
+
+		require.NoError(t, account.notes.SetTxNote("some-internal-tx-id", "some note, with a comma"))
+		fee := coin.NewAmountFromInt64(101)
+		require.Equal(t,
+			header+
+				",sent,123,satoshi,101,some-address,some-tx-id,\"some note, with a comma\"\n"+
```

Comment:
would've been nice to also test the timestamp.

Reply:
Done